src/Gui/Application.cpp

```diff
@@ -2112,6 +2113,8 @@ void Application::initOpenInventor()
     SoDB::init();
     SIM::Coin3D::Quarter::Quarter::init();
     SoFCDB::init();
+    SoFCDB::init();
```

Comment:
This line appears twice now. Please remove it as otherwise the app crashes with a failing assert()